cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt

```diff
@@ -0,0 +1,41 @@
+package com.cognifide.cogboard.config.controller
+
+import com.cognifide.cogboard.CogboardConstants
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonObject
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
+
+class ConfirmationSender(private val vertx: Vertx) {
+
+    fun confirmationAfter(action: (JsonObject) -> Boolean, body: JsonObject) =
```

Comment:
Maybe this format is better ?
``` 
fun confirmationAfter(action: (JsonObject) -> Boolean, body: JsonObject) =
            if (action(body))  sendOk()
            else sendError(body)
```

Reply:
In my opinion curly braces should be present here